### ISSUE TITLE
Fix UI colors

### ISSUE THREAD

**Issue by vimtaai** [rekaehreth/HKAdmin-service#49]
**Current Behavior**
UI has a very low contrast ratio right now, thanks to the usage of the dark text colors on the dark background.

**Expected Behavior**
The color palette should be updated so that the contrast ratio reaches an acceptable level and the text becomes easy to read. (Use lighter text on dark background)
